### DISCUSSION TITLE
5. Impfdosis

### DISCUSSION THREAD

**Discussion by TH-Pega** [Digitaler-Impfnachweis/certification-apis#242] [Q&A] [answered]
Hallo,

kann man gültige Impfzertifikate über die 5. ff Impfung ausstellen? Technisch geht es problemlos (RU), das BMG wollte es aber ausdrücklich nicht. Ärzte fragen bereits danach. 
_(Sinnvoller wäre natürlich, einfach eine weitere "Auffrischung" zu attestieren, statt irgendwann die 187. Impfung. Es zählt ja nur noch der zeitliche Abstand. Aber das ist ein größeres Thema.)_

**Comment by thinkberg** [comment, marked as answer]
Ist zwar etwas her, aber immer noch relevant. Fachlich kann ich dazu nichts sagen. Technisch ist es moeglich, aber die Apps muessen die Analyse (Bewertung) auch beherrschen.
